src/plugins/ui_stateful/edition.ts

```diff
@@ -692,6 +694,32 @@ export class EditionPlugin extends UIPlugin {
     this.colorIndexByRange = colorsToKeep;
   }
 
+  private addRowsForSpreadingFormula(content: string) {
+    if (!content.startsWith("=")) {
+      return;
+    }
+
+    try {
+      const evaluated = this.getters.evaluateFormula(this.sheetId, content);
+      if (!isMatrix(evaluated)) {
+        return;
+      }
+
+      const numberOfRows = this.getters.getNumberRows(this.sheetId);
+      const rowsInEvaluated = evaluated[0].length;
+      const missingRows = this.row + rowsInEvaluated - numberOfRows;
+      if (missingRows > 0) {
+        this.dispatch("ADD_COLUMNS_ROWS", {
+          sheetId: this.sheetId,
+          dimension: "ROW",
```

Comment:
rows but not columns ? :cry:

Reply:
columns ? never heard of them :roll_eyes: